e2e/plugins/index.ts

```diff
@@ -4,7 +4,7 @@
  */
 import cucumber from 'cypress-cucumber-preprocessor';
 
-module.exports = (on: Function) => {
+module.exports = (on: (a: unknown, b: unknown) => unknown) => {
```

Comment:
a and b?

Reply:
Linting complained that `Function` was not acceptable, so I was giving default var names for the type definition (but will add more sensible names)

Reply:
I ran into this too and went with `(on: (evt: string, callback: () => void) => void)`. Not a must change, but interesting to see what you came up with!

Reply:
https://github.com/cypress-io/cypress/blob/5c01832883e92de7a596e5d80a844cc04cf3ae32/cli/types/cypress.d.ts#L5002